firefox_puppeteer/tests/test_about_window.py

```diff
@@ -0,0 +1,91 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import By
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestAboutWindow(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.about_window = self.browser.open_about_window()
+
+    def tearDown(self):
+        try:
+            self.windows.close_all([self.browser])
+        finally:
+            FirefoxTestCase.tearDown(self)
+
+    def test_basic(self):
+        self.assertEqual(self.about_window.window_type, 'Browser:About')
+
+    def test_elements(self):
+        """Test correct retrieval of elements."""
+        self.assertNotEqual(self.about_window.dtds, [])
+
+        self.assertEqual(
+            self.about_window.deck.element.get_attribute('localName'),
+            'deck')
+
+        # apply panel
+        panel = self.about_window.deck.apply
+        self.assertEqual(panel.element.get_attribute('localName'), 'hbox')
+        self.assertEqual(panel.update_button.get_attribute('localName'), 'button')
+
+        # apply_billboard panel
+        panel = self.about_window.deck.apply_billboard
+        self.assertEqual(panel.element.get_attribute('localName'), 'hbox')
+        self.assertEqual(
+            panel.apply_billboard_button.get_attribute('localName'),
+            'button')
```

Comment:
Please fix the indentation by moving line 43 up behind the opening bracket. If the type has to wrap align it with `p`. Same for the other asserts here.